features/step_definitions/delivery_methods_steps.rb

```diff
@@ -36,7 +36,7 @@
 end
 
 Then(/^the customer should be notified by email$/) do
-  last_job = ActiveJob::Base.queue_adapter.enqueued_jobs.last
+  last_job = ActiveJob::Base.queue_adapter.enqueued_jobs.reverse.detect { |job| job[:job] == NotifyViaEmail }
```

Comment:
This feels like absolutely the wrong thing to do with cucumber. Is there a precedent for this style of testing here already?